src/foam/nanos/auth/ChangePasswordView.js

```diff
@@ -9,267 +9,166 @@ foam.CLASS({
   name: 'ChangePasswordView',
   extends: 'foam.u2.Controller',
 
-  documentation: `Change Password View: managing two states.
-  State one) ResetPassword:  redirect from email link, with a token to update forgotten password,
-  State two) UpdatePassword: simple change of password by a logged in user.
-  
-  Also manages two view types:
-  View one) Vertically aligned properties (this.horizontal = false),
-  View two) Horizontally aligned properties (this.horizontal = true)`,
+  documentation: 'renders a password change model',
 
   imports: [
-    'auth',
-    'notify',
-    'resetPasswordToken',
     'stack',
     'theme',
     'user'
   ],
 
   requires: [
-    'foam.nanos.auth.User',
-    'foam.u2.detail.SectionView'
+    'foam.u2.detail.SectionView',
+    'foam.nanos.auth.SendPassword',
```

Comment:
I'd like to see this get to a point where it doesn't need to know anything that is currently required. Challenge to you Jin

Reply:
you mean like we should not require 'foam.u2.detail.SectionView', 'foam.nanos.auth.SendPassword', etc., and make it work?